ms_graph.py

```diff
@@ -18,9 +18,6 @@ def generate_access_token(app_id, scopes):
         token_detail = json.load(open('ms_graph_api_token.json',))
         token_detail_key = list(token_detail['AccessToken'].keys())[0]
         token_expiration = datetime.fromtimestamp(int(token_detail['AccessToken'][token_detail_key]['expires_on']))
-        if datetime.now() > token_expiration:
```

Comment:
token expiration is only the current sort lived session token. wrong thing to check for expiry